simplified-app-simplye/src/main/assets/Accounts.json

```diff
@@ -1,6 +1,7 @@
 [
     {
         "authPasscodeLength": 4,
+        "cardCreatorUrl": "https://patrons.librarysimplified.org/",
```

Comment:
We still need special client-side code to handle this URL by launching the native-code card creator rather than a web view.

https://jira.nypl.org/browse/SIMPLY-1997 covers the server-side work necessary to allow a custom URI scheme here.